Formula/chakra.rb

```diff
@@ -0,0 +1,28 @@
+class Chakra < Formula
+  desc "The core part of the Chakra Javascript engine that powers Microsoft Edge"
+  homepage "https://github.com/Microsoft/ChakraCore"
+  url "https://github.com/Microsoft/ChakraCore/archive/v1.4.0.tar.gz"
+  sha256 "f6b4a3bce9e69bb3ea619a82805460906ef53d4f9da98b51e554dd48d6cdfdb3"
+
+  depends_on "cmake" => :build
+  depends_on "icu4c" => :optional
+
+  def install
+    if build.with? "icu4c"
+      icu4c = Formula["icu4c"]
+      icu = "--icu=#{icu4c.opt_include}"
```

Comment:
I'd populate an `args` array here instead as that's more typical formula form and inline `icu4c` as you're only using that variable once